modules/ROOT/examples/installation/post-installation-steps.sh

```diff
@@ -47,6 +70,23 @@ else
   upgrdcfg="n"
 fi
 
+# check if to upgrade an existing installation
+if [ "$do_upgrade" = "y" ]; then
+  read -p "Please specify the file to extract with full path: " -r -e answer
```

Comment:
I see that the convention you're using is to use `$answer` for storing user input, which makes perfect sense, but would it not make more sense to name the variable based on it's intended use?

Reply:
it is just the given answer.
more important is the variable having the name which highlights its itention.

Reply:
Fair point. It was worth raising.